packages/teams-js/src/public/runtime.ts

```diff
@@ -512,7 +512,10 @@ export function generateVersionBasedTeamsRuntimeConfig(
   Object.keys(mapVersionToSupportedCapabilities).forEach((versionNumber) => {
     if (compareSDKVersions(highestSupportedVersion, versionNumber) >= 0) {
       mapVersionToSupportedCapabilities[versionNumber].forEach((capabilityReqs) => {
-        if (capabilityReqs.hostClientTypes.includes(GlobalVars.hostClientType)) {
+        if (
+          GlobalVars.hostClientType !== undefined &&
```

Comment:
>GlobalVars.hostClientType !== undefined [](http://example.com/codeflow?start=10&length=39)

This unexpected case is now handled properly by skipping this code which would be meaningless if there was no `hostClientType` -- also many other things would blow up before this.